.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Main
+
+on:
+  push:
```

Comment:
**question**: Are we ever going to push to the master branch on this repo?

Reply:
probably not, but it shouldn't hurt anything for now.